SingularityService/src/main/java/com/hubspot/singularity/data/UsageManager.java

```diff
@@ -31,22 +32,30 @@
 
   private static final String SLAVE_PATH = ROOT_PATH + "/slaves";
   private static final String TASK_PATH = ROOT_PATH + "/tasks";
+  private static final String USAGE_SUMMARY_PATH = ROOT_PATH + "/summary";
```

Comment:
@ssalinas can you take a look at this piece? Just wanna make sure I got this set up right